src/gl/init.c

```diff
@@ -121,53 +129,61 @@ void initialize_gl4es() {
     env(LIBGL_XREFRESH, globals4es.xrefresh, "xrefresh will be called on cleanup");
     env(LIBGL_STACKTRACE, globals4es.stacktrace, "stacktrace will be printed on crash");
 
-
-    switch(ReturnEnvVarInt("LIBGL_FB")) {
-    	case 1:
+    const int LIBGL_FB_ENV_VAR = 
+	    ReturnEnvVarInt("LIBGL_FB");
+	const int LIBGL_FB_VALUE = 
+#       ifndef LIBGL_FB
+        LIBGL_FB_ENV_VAR
+#       else
+        0 == LIBGL_FB_ENV_VAR ? LIBGL_FB : 0
```

Comment:
I don't think that is correct. If you set LIBGL as a build parameter (and a value!=0), you will never be able to set LIBGL=0 using env. var.

Reply:
The code basically resolves to:
1. `LIBGL_FB_ENV_VAR` holds environmental variable or zero if env.var. is not set.
2.1 If library has been built without `LIBGL_FB` define than that's it.
2.2 If library has been built with `LIBGL_FB` define than if there is no env.var (LIBGL_FB_ENV_VAR is zero) than we use default LIBGL_FB value, else we use env.var. 

I see your point that if user has LIBGL_FB env.var. set to zero we still use `LIBGL_FB` define. But I do not know is this is really an issue or not. May be changed to check if env.var. exists in the first place and to no rely on zero result of `GetEnvVarInt`.

Reply:
Yes, there should be a test of the anv. var. existance, as 0 is usefull value.
also, it should be ` 0 == LIBGL_FB_ENV_VAR ? LIBGL_FB : LIBGL_FB_ENV_VAR` anyway I guess?

Reply:
so simply ` 0getenv("LIBGL_FB") ? LIBGL_FB : 0`

Reply:
> ...it should be ` 0 == LIBGL_FB_ENV_VAR ? LIBGL_FB : LIBGL_FB_ENV_VAR` anyway I guess?

Matter of taste, LIBGL_FB_ENV_VAR is zero anyway.




> Yes, there should be a test of the anv. var. existance, as 0 is usefull value. also...

Got it, it is an issue I will fix in a minute or so.

Reply:
Yeah, simply use that than!